extensions/raynab/src/components/transactions/transactionEditForm.tsx

```diff
@@ -106,148 +117,61 @@ export function TransactionEditForm({ transaction, forApproval = false }: Transa
     initialValues: {
       date: new Date(transaction.date),
       amount: formatToReadableAmount({ amount: transaction.amount, locale: false }),
-      payee_id: transaction.payee_id ?? undefined,
+      payee_id: payees?.some((p) => p.id === transaction.payee_id) ? (transaction.payee_id ?? '') : '',
```

Comment:
Breaking: This causes the initial value to always return an empty string as the `payees` list hasn't loaded yet. The easiest would be to revert to defaulting to the payee id in the existing transaction.
AFAIK there's no scenario where an existing transaction wouldn't have a payee.

```suggestion
      payee_id: transaction.payee_id ?? '',
```

Reply:
Something I tried to fix here and there is the edit form causes a lot of errors like this when loading.

21:57:40 The `value` provided to `Form.Dropdown` does not match any of its items.
            'b79e4a42-7f49-4136-8d84-1500161172db' is not a member of [].
21:57:40 The `value` provided to `Form.Dropdown` does not match any of its items.
            'b79e4a42-7f49-4136-8d84-1500161172db' is not a member of [].
21:57:40 The `value` provided to `Form.Dropdown` does not match any of its items.
            'b79e4a42-7f49-4136-8d84-1500161172db' is not a member of [].
21:57:40 The `value` provided to `Form.Dropdown` does not match any of its items.
            'b79e4a42-7f49-4136-8d84-1500161172db' is not a member of [].
21:57:40 The `value` provided to `Form.Dropdown` does not match any of its items.
            'b79e4a42-7f49-4136-8d84-1500161172db' is not a member of [].
21:57:41 The `value` provided to `Form.Dropdown` does not match any of its items.
            'b79e4a42-7f49-4136-8d84-1500161172db' is not a member of [].
21:57:41 The `value` provided to `Form.Dropdown` does not match any of its items.
            'b79e4a42-7f49-4136-8d84-1500161172db' is not a member of [].
21:57:41 The `value` provided to `Form.Dropdown` does not match any of its items.
            'b79e4a42-7f49-4136-8d84-1500161172db' is not a member of [].

Reply:
That's a great point. Checking for it is definitely the correct implementation, but it appears as if the dropdown tries to render the list despite the

Reply:
Maybe we can initiate the payees list with the initial transaction's payee data? That should fix the log:

```ts
const {
    data: payees = [{ id: transaction.payee_id ?? '', name: transaction.payee_name ?? '' }],
    isLoading: isLoadingPayees,
  } = usePayees(activeBudgetId || '')
```

Reply:
That worked!